src/jit/codegencommon.cpp

```diff
@@ -10588,6 +10628,45 @@ void CodeGen::genSetScopeInfoUsingsiScope()
 }
 #endif // USING_SCOPE_INFO
 
+#ifdef USING_VARIABLE_LIVE_RANGE
+void CodeGen::genSetScopeInfoUsingVariableRanges()
+{
+    VariableLiveKeeper* varLiveKeeper = compiler->getVariableLiveKeeper();
+
+    unsigned int varNum;
+    LclVarDsc*   varDsc;
+    unsigned int liveRangeIndex = 0;
+
+    for (varDsc = compiler->lvaTable, varNum = 0; varNum < compiler->info.compLocalsCount; varNum++, varDsc++)
```

Comment:
```C++
for (unsigned varNum = 0; varNum < compiler->info.compLocalsCount; varNum++)
{
    LclVarDsc* varDsc = compiler->lvaGetDesc(varNum);
```
I know that the JIT is full of this kind of contorted code that probably someone write in the '90 thinking that the compilers are dumb and need help. But that doesn't mean that new code has to continue the practice.

Reply:
What you say seems clearer. I am changing it.